### ISSUE TITLE
Wrong table name when using abstract models

### ISSUE THREAD

**Issue by rougeth** [treyhunner/django-simple-history#224]
Hello people,

What's the expected table name for a history model when I'm using an abstract model like this?

```
# core/models.py
class HistoryMixin(models.Model):
    history = HistoricalRecords(inherit=True)
    class Meta:
        abstract = True

class SameModuleModel(HistoryMixin, models.Model):
    pass

# app1/models.py
class AnotherAppModel(HistoryMixin, models.Model):
    pass
```

In this example, the `makemigrations` output is:

```
Migrations for 'core':
  0001_initial.py:
    - Create model HistoricalAnotherAppModel
    - Create model HistoricalSameModuleModel
    - Create model SameModuleModel
Migrations for 'app1':
  0001_initial.py:
    - Create model AnotherAppModel
```

All models that inherit from `HistoryMixin` will have its history db table named like `core_historicalmodel_name` and not like `model_module_using_HistoryMixin_historicalmodel_name`.

Am I not seeing something here?

Thanks.

**Comment by gabn88** [comment]
If I use the registration approach (register(AbstractAddress, inherit=True)), I get

    from address.abstract_models import AbstractAddress
  File "address/abstract_models.py", line 175, in <module>
    register(AbstractAddress, inherit=True)
  File "/site-packages/simple_history/__init__.py", line 33, in register
    records.finalize(model)
  File "/site-packages/simple_history/models.py", line 88, in finalize
    history_model = self.create_history_model(sender)
  File "/site-packages/simple_history/models.py", line 116, in create_history_model
    app = apps.app_configs[model._meta.app_label]
KeyError: 'address'

My Meta is:
    class Meta:
        abstract = True
        verbose_name = _('Address')
        verbose_name_plural = _('Addresses')


Or are abstract models not supported?

**Comment by rougeth** [comment]
@gabn88 could you show us more details about your case? I don't see any relation with the issue I presented.

**Comment by macro1** [comment]
@rougeth: This looks like an edge case that was missed when the feature was added. I'll look at patching it.

Several people asked for the feature, so even though I don't use it myself I put some code together that would make it work, and asked for testing in addition to the unit tests I added, but I guess it was missed.

Hopefully having the table names changed won't cause too many issues for people already using the feature.

**Comment by rougeth** [comment]
@macro1 I read the issues about this feature and I can't see a case that will cause issues when fixing it.

I'll try to write a PR in the next few days.

**Comment by melinath** [comment]
I just ran into this as well. I guess for the time being I'll just define history on every child model as well....

**Comment by rougeth** [comment]
@tirtasantoso I would do what @melinath said, just define the history field on each child.

**Comment by rossmechanic** [comment]
This issue is fixed in #324 with tests

**Comment by rossmechanic** [comment]
This issue is fixed for simple-history versions >=1.9.1
